frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -401,9 +401,207 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
   }
 
   /**
-   * Edit challenge overview function
+   * Edit challenge overview with file function
+   */
+  editChallengeOverviewUpload() {
+    const SELF = this;
+    SELF.apiCall = (params) => {
+      const FORM_DATA: FormData = new FormData();
+      FORM_DATA.append('overview_file', params['overview_file']);
+      SELF.apiService
+        .patchFileUrl(
+          SELF.endpointsService.editChallengeDetailsURL(SELF.challenge.creator.id, SELF.challenge.id),
+          FORM_DATA
+        )
+        .subscribe(
+          (data) => {
+            SELF.challenge.description = data.description;
+            SELF.globalService.showToast('success', 'The description is successfully updated!', 5);
```

Comment:
Please change the message to `Challenge description updated successfully!`

Reply:
Addressed.